test/integration/targets/ansible-galaxy-collection/tasks/upgrade.yml

```diff
@@ -0,0 +1,166 @@
+##### Updating collections with a new version constraint
+
+# No deps
+
+- name: reset installation directory
+  file:
+    state: "{{ item }}"
+    path: "{{ galaxy_dir }}/ansible_collections"
+  loop:
+    - absent
+    - directory
+
+- name: install a collection
+  command: ansible-galaxy collection install namespace1.name1:0.0.1 -s pulp_v3 {{ galaxy_verbosity }}
```

Comment:
Minor nit but do we need the `-s`, seems like we could simplify the tests by just selecting the first server in the `ansible.cfg` and only override if we need to.